Source/SA/RFmx LTE/RFmxLTE.cs

```diff
@@ -201,6 +199,22 @@ public static void ConfigureModAcc(RFmxLteMX lte, ModAccConfiguration modAccConf
             lte.ModAcc.Configuration.ConfigureSynchronizationModeAndInterval(selectorString, modAccConfig.SynchronizationMode, modAccConfig.MeasurementOffset, modAccConfig.MeasurementLength);
             lte.ModAcc.Configuration.ConfigureEvmUnit(selectorString, modAccConfig.EvmUnit);
         }
+        public static void SelectAndInitiateMeasurements(RFmxLteMX lte, RFmxLteMXMeasurementTypes[] measurements, AutoLevelConfiguration autoLevelConfig,
+            bool enableTraces = false, string selectorString = "", string resultName = "")
+        {
+            // Aggregate the selected measurements into a single value
+            // OR of 0 and x equals x
+            RFmxLteMXMeasurementTypes selectedMeasurement = 0;
+            foreach (RFmxLteMXMeasurementTypes measurement in measurements)
+                selectedMeasurement |= measurement;
+            lte.SelectMeasurements(selectorString, selectedMeasurement, enableTraces);

```

Comment:
I'm wondering, what if the array is empty? I really don't see people using this function to disable measurements. 
Could we consider not touching SelectMeasurements if the array is empty? That might muddle understanding a little though. I'm just considering ways to still use this function with the other measure functions.

Reply:
My desire would be that the user would receive an error if they have not selected any measurements. Immediately after Initiate will be called, which will fail with nothing selected. In other words, no selection should be considered a user error. 

I'd be happy leaving whatever default error will be thrown by RFmx.